### DISCUSSION TITLE
Hebrew Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#1073] [Language Leaders]
@leranp @haim-b when you get a chance, can you please look into translating the sentences for `HassGetState`?

**Comment by leranp** [comment]
Yes, i will try to do it today

**Reply by haim-b** [reply]
Already started and have a pull request.

**Reply by synesthesiam** [reply]
Thanks!

**Comment by leranp** [comment]
Done
